src/transactions/test/SorobanTxTestUtils.cpp

```diff
@@ -430,46 +430,64 @@ ContractInvocationTest::getTTLEntryWriteFee()
 }
 
 int64_t
-ContractInvocationTest::getRentFeeForExtension(LedgerKey const& key,
-                                               uint32_t newLifetime)
+ContractInvocationTest::getRentFeeForExtension(
+    xdr::xvector<LedgerKey> const& keys, uint32_t newLifetime)
 {
-    size_t entrySize = 0;
-    uint32_t extendTo = 0;
+    int64_t rentFee = 0;
+    int64_t ttlBytes = 0;
+    size_t numExtensions = 0;
+
+    for (auto const& key : keys)
     {
         LedgerTxn ltx(mApp->getLedgerTxnRoot());
-        auto ledgerSeq = ltx.getHeader().ledgerSeq;
+        auto ledgerSeq =
+            ltx.getHeader().ledgerSeq; // Add one to simulate a bump in the
```

Comment:
Nit: I don't think we add one anymore. remove this comment.

Reply:
Removed the comment.

Reply:
Cool, I think test tx meta changed since we have different fees, but once you push that the CI should pass and we can merge.